lib/msal-node/src/request/ManagedIdentityRequest.ts

```diff
@@ -12,4 +12,7 @@ import { ManagedIdentityRequestParams } from "./ManagedIdentityRequestParams.js"
  * - resource  - resource requested to access the protected API. It should be of the form "{ResourceIdUri}" or {ResourceIdUri/.default}. For instance https://management.azure.net or, for Microsoft Graph, https://graph.microsoft.com/.default
  */
 export type ManagedIdentityRequest = ManagedIdentityRequestParams &
-    CommonClientCredentialRequest;
+    CommonClientCredentialRequest & {
+        clientCapabilities?: Array<string>;
+        accessTokenSha256Hash?: string;
```

Comment:
Consider renaming this to `revokedToken`

Reply:
done